DesignKit/Source/Fonts.swift

```diff
@@ -0,0 +1,46 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import UIKit
+
+/// Describe Fonts
+@objc public protocol Fonts {
```

Comment:
We should have an exhaustive list of all fonts described in Figma. Thus, people will not try to make unexpected customisations.

There are 20 fonts defined on figma for ios. 10 more than for web and android. We need to be sure we need all.